spec/inputs/jdbc_spec.rb

```diff
@@ -1042,4 +1044,38 @@
       end
     end
   end
+
+  context "when fetching Various Typed data" do
+
+    let(:settings) do
+      {
+      "statement" => "SELECT * from types_table"
+      }
+    end
+    let(:num_rows) { 1 }
```

Comment:
not sure how necessary this is since it is `1`. then the loop in the `before` block can be removed.